### PR TITLE
MF-792:Patient header: Reduce space between borders and sectional headings

### PR DESCRIPTION
**Pull request by maureennduta**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
Reduce space between borders and sectional headings according to this [design](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/60d5db4ef3058bbcafbc8ad1)

## Screenshots

image

## Related Issue
Issue [MF-792](https://issues.openmrs.org/browse/MF-792)

### CODE REVIEW

**Comment by HerbertYiga** [comment]
looks good to me thanks @maureennduta  cc @brandones

File: packages/esm-patient-banner-app/src/contact-details/contact-details.scss [hunk 1]
```diff
@@ -18,7 +18,8 @@
 
 .row .col {
   flex: 1;
-  padding: 1.3125rem;
+  padding: 1.0125rem;
+  padding-top: 0.15rem;
```
Comment (denniskigen) [inline packages/esm-patient-banner-app/src/contact-details/contact-details.scss:22]: You probably want to condense this into one declaration using the padding property: `padding: padding-top-value padding-right-value padding-bottom-value padding-left-value` . As currently written, it's not entirely clear what amounts of padding should be applied.

Comment (maureennduta) [reply packages/esm-patient-banner-app/src/contact-details/contact-details.scss:22]: Ok, I had used rem to strive for uniformity but I will update this.

**Comment by denniskigen** [comment]
Hey @maureennduta, sorry this hasn't been addressed for so long. Could you fix the huge diff - looks like a rebase gone bad.

**Comment by denniskigen** [comment]
Thanks, @maureennduta. Closing because this got fixed in https://github.com/openmrs/openmrs-esm-patient-chart/pull/442.

**Pull request closed without merging**